apps/src/lab2/projects/ProjectManager.ts

```diff
@@ -248,6 +244,11 @@ export default class ProjectManager {
     this.publishHelper(false);
   }
 
+  async getVersionList() {
+    // TODO: Error handling
```

Comment:
Just checking, do we have a JIRA for this?

Reply:
yep! https://codedotorg.atlassian.net/browse/CT-712